### PR TITLE
#33: relax the constraints of floor_sum

### PR DESCRIPTION
**Pull request by yosupo06**
71ms: https://atcoder.jp/contests/practice2/submissions/19500203

This change doesn't degenerate the performance from #88

### CODE REVIEW

File: atcoder/internal_math.hpp [hunk 1]
```diff
@@ -177,6 +177,35 @@ constexpr int primitive_root_constexpr(int m) {
...
+        if (a >= m) {
+            ans += n * (n - 1) / 2 * (a / m);
+            a %= m;
+        }
+        if (b >= m) {
+            ans += n * (b / m);
+            b %= m;
+        }
+
+        unsigned long long y_max = a * n + b;
```
Comment (rsk0315) [inline atcoder/internal_math.hpp:198]: If `y_max = a * n + b >= 2^64`, are `y_max / m` and `y_max % m` calculated properly?

Comment (yosupo06) [reply atcoder/internal_math.hpp:198]: a * n + b <= (m - 1) * n + (m - 1) <= (m - 1) * (n + 1) <= (2 ^ 32 - 2) * 2 ^ 32 < 2 ^ 64

Comment (rsk0315) [reply atcoder/internal_math.hpp:198]: Aha! I forgot the constraints about `m`, sorry.

**Comment by yosupo06** [comment]
fix #33

**Pull request merged**